### PR TITLE
Get images function

### PR DESCRIPTION
**Pull request by nkorinek**
Added a method named `get_plot_image()` that will return the images stored in a matplotcheck RasterTester obejct, and tests for that function. @lwasser

### CODE REVIEW

**Comment by nkorinek** [comment]
@lwasser should be good to merge!

**Comment by lwasser** [comment]
@nkorinek to clarify this is a method correct? :) just want to ensure we are keeping terms straight... but i'm excited to have a look at it... heading off to class now!

**Comment by nkorinek** [comment]
@lwasser good catch, it is a method, not a function...oops! Updated the comment to reflect that

File: matplotcheck/raster.py [hunk 1]
```diff
@@ -163,19 +163,39 @@ def assert_legend_accuracy_classified_image(
 
         ### IMAGE TESTS/HELPER FUNCTIONS ###
 
+    def get_plot_image(self):
+        """Returns images stored on the Axes object as a lsit of numpy arrays.
```
Comment (lwasser) [inline matplotcheck/raster.py]:
```suggestion
        """Returns images stored on the Axes object as a list of numpy arrays.
```

**Comment by lwasser** [comment]
@nkorinek merging this!!! thank you!

**Pull request merged**